bluemira/equilibria/fem_fixed_boundary/equilibrium.py

```diff
@@ -254,8 +269,18 @@ def solve_transport_fixed_boundary(
     directory = get_bluemira_path("", subfolder="generated_data")
     mesh_name_msh = mesh_filename + ".msh"
 
-    parameterisation = plasma_parameterisation(
-        {k: {"value": v} for k, v in asdict(params).items()}
+    paramet_params = PlasmaFixedBoundaryParams(
+        **{
+            k: v
+            for k, v in zip(
+                parameterisation.variables.names, parameterisation.variables.values
+            )
+            if k in PlasmaFixedBoundaryParams.__annotations__.keys()
```

Comment:
Doesn't really matter in this case, but I recently read a suggestion that it's better practice to use `[f.name for f in fields(DataClass)]`, than `__annotations__.keys()`, as `__annotations__` will not include inherited members.

Again, not an issue in this case, as `PlasmaFixedBoundaryParams` is not inheriting from anything, but thought I'd share.